src/lib/route.js

```diff
@@ -36,11 +36,13 @@ exports.getRouteHandlers = function (parsedUrl, action, queryParams) {
                 logger.logHttpRequest(message, httpRequest);
 
                 this.response.headers.forEach(function (header) {
-                    res.set(header.name, header.value);
```

Comment:
`set` is an Express method that automatically processes added headers; `setHeader` is an inherited Node method that does not do any processing.